bot/exts/utils/snekbox.py

```diff
@@ -568,3 +586,7 @@ def predicate_emoji_reaction(ctx: Context, reaction: Reaction, user: User) -> bo
 async def setup(bot: Bot) -> None:
     """Load the Snekbox cog."""
     await bot.add_cog(Snekbox(bot))
+
+    guild = Object(id=constants.Guild.id)
```

Comment:
I think that this code should go away once [this PR](https://github.com/python-discord/bot-core/pull/171) is completed